docs/core/idioms/notifications.md

```diff
@@ -0,0 +1,56 @@
+# Configuring notifications
+
+Sending notifications based on events in Prefect is a common pattern. Types of notifications generally include functionality such as posting state of runs to Slack, sending a text message on failure, etc. There are many places in Prefect to hook in notification systems and this idiom will go over three possibilities.
+
+### Downstream tasks
+
+Adding tasks to your flow that respond to upstream events is one way of sending notifications. These tasks can be built in any way you would like and the Prefect task library even ships with a few [notification tasks](/api/latest/tasks/notifications.html)! Below is an example of using the [`SlackTask`](/api/latest/tasks/notifications.html#slacktask) to send a notification to Slack.
```

Comment:
I think we should call out using `triggers` as a way of only firing the notification in certain situations. (e.g., only notify me when the task fails via an `any_failed` trigger on the slack task)